CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-# v 1.3.0 (?)
+# v 1.2.2 (?)
```

Comment:
So, as discussed on slack, I would undo this change. Keep the change entry, but revert the change to the version here.

Reply:
oeps, sorry, bad advice from my side.